### ISSUE TITLE
Symbolic links in mounted archive are not shown in Dolphin

### ISSUE THREAD

**Issue by W0nderer** [borgbackup/borg#2317]
If an archive containing symbolic links is mounted then the links do not appear in Dolphin, even though they are shown in an `ls` of the mounted directory.  This can be shown by running the example below.

```
# Set path to your repository
repo=/path/to/repository

# Create: directories, files, and some links to be archived.
mkdir -p ~/test-borg
cd ~/test-borg
touch f1
ln -s f1 l1 # Relative, to file, at top.
mkdir d
ln -s d ld  # Relative, to dir, at top.
touch d/f2
ln -s $PWD/d/f2 d/l2 # Absolute, to file, in sub-dir.

# Create repository and archive.
mkdir -p $repo
borg init -e none $repo
borg create -v $repo::Test-slinks .

# Mount archive.
mkdir -p -m 777 /mnt/borg
borg mount -v $repo::Test-slinks /mnt/borg

# List mounted archive
ls -lR /mnt/borg     # All links are listed.

# Open Dolphin
dolphin /mnt/borg & # No links are shown.
```
This occurs running borg-linux64_1.0.10 in openSUSE 42.1 with Dolphin 15.12.3 in a KDE 5.5.5 Plasma desktop, also in openSUSE 42.2 with Dolphin 16.08.2 in a KDE 5.8.3 Plasma desktop.  Both the filesystems are Ext4 with ACL enabled, and /home has Extended User Attributes enabled.

I don't know if this is a bug in Borg or Dolphin.  But if the latter what is different about the way Borg mounts the archive that makes Dolphin not show links?

**Comment by enkore** [comment]
Are the link targets valid? If not, Dolphin may have elected to not show dangling symlinks.

**Comment by W0nderer** [comment]
Thanks for trying to reproduce this.  I have just tried in an old openSUSE 13.1 with KDE/Dolphin 4.11.5 - all the links were shown in Dolphin.  So it looks like it's KDE/Dolphin 5 problem, it would be helpful if you or someone else can confirm this.

**Comment by enkore** [comment]
With dolphin 16.12.2-1 the links show up.

**Comment by W0nderer** [comment]
openSUSE 42.1 has: kio 5.21.0-23.2, fuse 2.9.3-8.2.
openSUSE 42.2 has: kio 5.26.0-6.2,   fuse 2.9.3-9.6

In both releases of oS fuse 2.9.3 is fairly old (May 2015), could this be relevant?

The links show OK in Konqueror, but that is still using KDE4 in both openSUSEs.

Thanks for checking it in Arch, seems we really need someone to verify it in an openSUSE Leap 42.

**Comment by W0nderer** [comment]
I have been investigating to see if it's a more general problem of using Dolphin/KDE5 with FUSE in openSUSE, and have also found that it doesn't display the links when archivemount uses FUSE to mount a tar archive.  

As the problem doesn't seem to be due to Borg I will report the more general problem on the openSUSE forum.  Thanks for your help.

**Comment by rdmtinez** [comment]
I've just experienced the same problem on Ubuntu 18.04.2
I was on my workplaces server and created a SymLink from a folder on my $HOME to a shared location our group uses. I can see the SymLink where I created it but only using the command line, it doesn't show up on Nautilus.

Furthermore, when I tried to redo the SymLink--first by deleting the symlink:
`rm -r SymLink`
it ended up deleting the files IN the Orginal folder, but the the symlink itself was not deleted, however before I noticed those files were gone I got a message saying "SymLink is not a directory", so then I tried
`rm SymLink`
and got "SymLink is a directory".

Next I unliked the folder:
`unlink SymLink` 
and that was able to remove the link, but my files are still gone :(
